src/common/api/types.ts

```diff
@@ -570,7 +575,6 @@ export interface Cpu {
   flags?: string[];
   modelName?: string;
   architecture?: string;
-  clockMegahertz?: number;
```

Comment:
Quite recently there was this commit that fixed a bug for ACM, so I think the field `clockMegahertz` is necessary (as well as keeping the usage in `getHumanizedCpuClockSpeed`)

https://github.com/openshift-assisted/assisted-ui-lib/commit/c50a119e67b030c6a4188e3fedbf3ed6a111cafb

Reply:
@celdrake this removal comes from a change in the swagger.yaml. The BE removed this field without being aware if we were consuming it. 
cc @osherdp

Reply:
Seems like it has been changed with some kind of forward-compatibility handling: https://github.com/openshift/assisted-service/blob/master/api/v1beta1/agent_types.go#L101

Meaning you checked first for ``frequency`` and if it doesn't then using ``clockMegahertz``
So I think there was some kind of awareness about this field name migration

Reply:
10x @osherdp so I guess we should use just `frequency` now. Is this correct?

Reply:
@nmagnezi can you confirm? ^
I know it's an issue from more than a year ago, but should they use ``frequency`` instead of ``clockMegahertz`` as written [here](https://github.com/openshift/assisted-service/blob/master/api/v1beta1/agent_types.go#L101)?

Reply:
If you are asking about REST-API, yes.
The KubeAPI agent controller also does read it from `frequency`.

https://github.com/openshift/assisted-service/blob/ddb6400f03c15526c39fa6a2b1bc1f1b7e6def92/internal/controller/controllers/agent_controller.go#L900-L908